### DISCUSSION TITLE
Rewrite syntax

### DISCUSSION THREAD

**Discussion by trapd00r** [beetbox/beets#3963] [Q&A]
![img](https://i.imgur.com/9rQyjuE.png)

How would I go about fixing issues like this? The compilations are named slightly different on musicbrainz beginning with Dream Dance 20. I would like all of them named like this:

Dream Dance 0n

where, if n < 10, n should be padded with a zero.

I am not sure how I would go about adding a rewrite rule when I need to take elements of the name into account and not doing a simple replace.

**Comment by wisp3rwind** [comment]
That should be doable using the [`inline` plugin](https://beets.readthedocs.io/en/stable/plugins/inline.html), writing a small Python snippet to do the replacement.
